Formula/r/recc.rb

```diff
@@ -0,0 +1,196 @@
+class Recc < Formula
+  desc "Remote Execution Caching Compiler"
+  homepage "https://buildgrid.gitlab.io/recc"
+  url "https://gitlab.com/BuildGrid/buildbox/buildbox/-/archive/1.2.20/buildbox-1.2.20.tar.gz"
+  sha256 "5ad5ccbad1250accc984823bd5d78604441c7711320b0569889f45f329ed6311"
+  license "Apache-2.0"
+  head "https://gitlab.com/BuildGrid/buildbox/buildbox.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "tomlplusplus" => :build
+  depends_on "abseil"
+  depends_on "c-ares"
+  depends_on "glog"
+  depends_on "grpc"
+  depends_on "openssl@3"
+  depends_on "protobuf"
+  depends_on "re2"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "gflags"
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "util-linux" => :build
```

Comment:
```suggestion
    depends_on "util-linux"
```
Looks like you need this at runtime.